src/tests/runTest.ts

```diff
@@ -18,13 +18,34 @@
 // All tests ran here are ones that require the vscode API
 import * as path from 'path'
 import * as cp from 'child_process'
+import { parseArgs } from 'util'
 import {
   runTests,
   resolveCliArgsFromVSCodeExecutablePath,
   downloadAndUnzipVSCode,
 } from '@vscode/test-electron'
 
 async function main() {
+  const {
+    values: { disable_cert_verification },
+  } = parseArgs({
+    options: {
+      disable_cert_verification: {
+        type: 'boolean',
+        short: 'k',
+        default: false,
+      },
+    },
+  })
+  if (disable_cert_verification) {
+    process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0'
+  }
+
+  // The version of VS Code to use for running the test suite in
+  const testVsCodeVersion = process.env.TEST_VSCODE_VERSION
+    ? process.env.TEST_VSCODE_VERSION
+    : 'stable'
```

Comment:
@stevedlawrence, allow the VS Code version where we run the test suite to be externally configurable, with the latest stable release as the default.

Reply:
:+1: I like it. Though, I would just suggest we default to a pinned version and not stable, and some nightly CI can change this environment variable to stable or nightly to ensure we work with latest releases. So tests are still pinned but we still have a way to ensure we work with the latest and greatest.